From: Sakura286 <sakura286@outlook.com>
Date: Sun, 16 Mar 2025 16:00:21 +0800
Subject: fix riscv64 build

---
 src/include/gdrwrap.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/gdrwrap.h b/src/include/gdrwrap.h
index 669cbac..6165e6a 100644
--- a/src/include/gdrwrap.h
+++ b/src/include/gdrwrap.h
@@ -41,7 +41,7 @@ static inline void wc_store_fence(void) { asm volatile("sync") ; }
 #elif defined(__x86_64__)
 #include <immintrin.h>
 static inline void wc_store_fence(void) { _mm_sfence(); }
-#elif defined(__aarch64__)
+#elif defined(__aarch64__) || (defined(__riscv) && __riscv_xlen == 64)
 #ifdef __cplusplus
 #include <atomic>
 static inline void wc_store_fence(void) { std::atomic_thread_fence(std::memory_order_release); }
